pkg/gateway/client/mcpauditlog.go

```diff
@@ -43,8 +43,11 @@ func (c *Client) GetMCPAuditLogs(ctx context.Context, opts MCPAuditLogOptions) (
 		}
 
 		var userIDs []string
+		// Don't lowercase the query for the selecting from the database. Some databases aren't case-sensitive,
+		// and we want to preserve the case of the search term.
```

Comment:
This comment seems to contradict what we are doing? It says don't lowercase, but we are calling `strings.ToLower`

Reply:
It was confusing. I made a code-change to hopefully make it clearer.